chainextensions/types/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "chain-extension-types"
```

Comment:
What is the purpose of having types package?

My opinion is all chainextensions (including types) should be independent of each other. So, it's better to put types in separate packages, rather than all chainextensions' types in a single package.
Because different chains have a different combination of pallets. Substrate based chains should be able to pick one chainextension even if it doesn't support another supported in astar-frame.

Reply:
Rust will compile crate as a unit. If you put all types (dapps-stakings' types, pallet-a's types, pallet-b's types) in one single package, unnecessary types will be compiled too, that's not good.
Assume ChainX support pallet-a chainextension, it will compile dapps-staking types too even if it doesn't need them.

Reply:
_it's better to put types in separate packages, rather than all chainextensions' types in a single package._
Totally agree - I'll create a dapps-staking-chain-extension-types crate and only expose the relevant types